build.sbt

```diff
@@ -1258,7 +1276,7 @@ lazy val root = (project in file("."))
       tagRelease,
       releaseStepCommandAndRemaining("+publishSigned"),
       releaseStepCommand("dist/Universal/packageZipTarball"),
-      releaseStepCommand("kafkaBaseEngineRuntime/Universal/packageZipTarball"),
+      releaseStepCommand("liteKafkaEngineRuntime/Universal/packageZipTarball"),
```

Comment:
I think we also need Docker/publish here?
Also, we probably want to publish image in preview branch build? pr.yml, link 579?